api/tests/unit/infrastructure/validators/grecaptcha-validator_test.js

```diff
@@ -53,6 +53,7 @@ describe('Unit | Service | google-recaptcha-validator', () => {
 
       it('should return a rejected promise when user response token is invalid', function() {
```

Comment:
je renommerai en "should log an error and return a rejected promise when user response token is invalid". peut-être aussi séparer le test en 2 : le rejet en erreur et le log. à toi de voir.